### DISCUSSION TITLE
微软翻译超时，网络连接正常

### DISCUSSION THREAD

**Discussion by myg321** [YiiGuxing/TranslationPlugin#5278] [General]
翻译好像用不了了，不论是关梯子还是开梯子，在网络连接的情况下，翻译都超时且无法显示，切换成微软翻译也不行

**Comment by YiiGuxing** [comment]
之个你只能检查一下你的网络环境了，看看IDE是不是配置代理，系统全局代理这些。

**Reply by myg321** [reply]
确实是IDE代理的问题，感谢指点，有类似问题的友友可以参考这篇帖子解决：https://blog.csdn.net/vxstar/article/details/131845337
